Packs/DigitalShadows/README.md

```diff
@@ -44,18 +44,59 @@ Intelligence Incidents
 Supplier/Vendor Monitoring
 Vulnerability/Exploit Monitoring
 
-## What makes Digital Shadows unique
+## What makes ReliaQuest GreyMatter DRP unique
 
 - Exposure monitoring  
-  Digital Shadows monitors for the exposure of these assets across the open, deep, and dark web. This includes code-sharing sites, file-sharing sites, criminal forums, chat messages, social media, and search engines. For additional enrichment and incident response, users can search across many of these sources via "Shadow Search". Furthermore, our Collections and Closed Sources teams focus on continually adding to these sources.
+  ReliaQuest GreyMatter DRP monitors for the exposure of these assets across the open, deep, and dark web. This includes code-sharing sites, file-sharing sites, criminal forums, chat messages, social media, and search engines. For additional enrichment and incident response, users can search across many of these sources via "Shadow Search". Furthermore, our Collections and Closed Sources teams focus on continually adding to these sources.
 
 - Risk Identification
   Our technology reduces the mentions of these assets to only those instances that constitute a risk to your business. On average, this combination of technology and human analysis removes more than 95% of the total mentions of an organization's assets. Each alert has a risk score, ensuring your team is not overwhelmed by irrelevant mentions and can easily prioritize alerts.
 
 - Take Action and Protect  
   Alerts provide you all the context you need to make quicker, better decisions. Users also benefit from playbooks for remediating the risk, including the ability to launch takedowns.
 
-Email: info@digitalshadows.com
+## Configuration Guide
+
+## Request Digital Shadows API Credentials
+
+To use the application you will need to request an API Key and secret from Digital Shadows Support. Email support@digitalshadows.com stating that you would like to utilize the Digital Shadows Cortex XSOAR Integration and your SearchLightTM account details to have a new API Key created and assigned to you. 
+
+To find your SearchLightTM  account details; in the SearchLightTM  portal please navigate to: 
+- ‘Learn’ > ‘API Documentation’  
+- Use the left hand filter to select ‘Keywords’   
+- Scroll down to ‘Account’ and the ID is displayed
+
+## Configuration
```

Comment:
This configuration information should actually be in the integration description doc, `Packs/DigitalShadows/Integrations/ReliaQuestGreyMatterDRPIncidents/ReliaQuestGreyMatterDRPIncidents_description.md`.

Reply:
will add it there